modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -333,7 +336,7 @@ class ONNXWithRemap : public ONNXtest {
             const auto out_size = std::min(out_onnx.at(i).total(), out_gapi.at(i).total());
             for (size_t d_idx = 0; d_idx < out_size; ++d_idx) {
                 // If both outputs used remapping, they have an end-of-data mark
-                if ((gp[d_idx] == -1.f) && (op[d_idx] == -1.f)) {
+                if (gp[d_idx] == -1.f) {
```

Comment:
May be `||`?
Is `gp` always smaller?

Reply:
-1 is end of data mark. We check only gapi data.

Reply:
I agree. But this validation returns true even if in fact there are different amount of objects in ONNX and G-API outputs. Is it correct?

Reply:
Yes. All tested networks with remap have positive outputs.